operator/clusterutilities.go

```diff
@@ -43,12 +43,30 @@ const AFFINITY_NOTINOperator = "NotIn"
 const DefaultArchiveTimeout = "60"
 const DefaultPgHaConfigMapSuffix = "pgha-default-config"
 
+// affinityType represents the two affinity types provided by Kubernetes, specifically
+// either preferredDuringSchedulingIgnoredDuringExecution or
+// requiredDuringSchedulingIgnoredDuringExecution
+type affinityType string
+
+const (
+	requireScheduleIgnoreExec affinityType = "requiredDuringSchedulingIgnoredDuringExecution"
+	preferScheduleIgnoreExec   affinityType = "preferredDuringSchedulingIgnoredDuringExecution"
```

Comment:
Whitespace looks off. Might need `go fmt` on this file.

Reply:
Formatting has been fixed